Libraries/Modal/Modal.js

```diff
@@ -25,12 +25,6 @@ import type EmitterSubscription from '../vendor/emitter/EmitterSubscription';
 import RCTModalHostView from './RCTModalHostViewNativeComponent';
 import {VirtualizedListContextResetter} from '../Lists/VirtualizedListContext.js';
 
-/**
- * The Modal component is a simple way to present content above an enclosing view.
- *
- * See https://reactnative.dev/docs/modal.html
- */
-
```

Comment:
Moved the updated comments directly above the declared component so that it can be parsed by react-docgen.